db/structure.sql

```diff
@@ -168,16 +179,16 @@ CREATE TABLE `lighthouse_sample` (
   `filtered_positive` tinyint(1) DEFAULT NULL COMMENT 'Filtered positive result value',
   `filtered_positive_version` varchar(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Filtered positive version',
   `filtered_positive_timestamp` datetime DEFAULT NULL COMMENT 'Filtered positive timestamp',
-  `lh_sample_uuid` varchar(36) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Sample uuid created in crawler',
-  `lh_source_plate_uuid` varchar(36) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Source plate uuid created in crawler',
+  `lh_sample_uuid` varchar(36) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Sample uuid created in crawler',
```

Comment:
Do you know why these things have changed?

Reply:
These changes were made automatically when I called `bundle exec rails db:migrate` to apply the view migration

Reply:
Understood.